drivers/digit7seg/digit7seg.c

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2024 Orange
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_digit7seg
+ * @file
+ * @brief       Device driver for less than 5 digits of 7 segments without IC
```

Comment:
```suggestion
 * @brief       Device driver for up to 4 digits of 7 segments without IC
```

do we really want such restriction?

Reply:
Yes, most 7-digit segments are composed of 4 digits, which is easier to manage.